doc/system_requirements.md

```diff
@@ -19,111 +19,160 @@ Goals of this plugin:
 
 ## Features
 
-Features are the highest level requirements in this document that describe the main functionality of ETC.
+Features are the highest level requirements in this document that describe the main functionality of PK.
 
 ### Verify Repository
 
 `feat~verify-repo-setup~1`
 
-This plugin can verify if the repository setup.
+PK can check if the repository structure is up to date.
 
 Needs: req
 
-### Fix Repository
+#### Verify POM File
 
-`feat~fix-repo-setup~1`
-
-This plugin can fix the repository setup.
+`req~verify-pom-file~1`
 
 Needs: req
 
-## Requirements
-
-### Defined Repository Structure
+##### Verify Maven Plugins
 
-`req~repository-structure~1`
+`req~verify-mvn-plugins~1`
 
-We define the structure of the repositories by implementing this plugin. That means that each change of the template structure is rolled out over a new release of this plugin.
+PK can verify that a project's pom.xml file contains certain plugins.
 
 Covers:
 
-* [feat~verify-repo-setup~1](#verify-repository)
-* [feat~fix-repo-setup~1](#fix-repository)
+* `req~verify-pom-file~1`
 
 Needs: dsn
 
-### Convenient Repository Structure Definition
+##### Verify Maven Dependencies
 
-`req~convenient-repository-structure~1`
+`req~verify-maven-dependencies~1`
 
-Since the template is updated often and by different team members, the way, in that we define the template inside of this plugin, must be convenient and easily adaptable.
+PK can verify that a project's pom file contains certain mandatory dependencies.
 
 Covers:
 
-* [feat~verify-repo-setup~1](#verify-repository-structure)
-* [feat~fix-repo-setup~1](#fix-repository-structure)
+* `req~verify-pom-file~1`
 
 Needs: dsn
 
-### Checking Repository Structure During Build
+#### Verify changes_x.x.x.md file
 
-`req~verify-repo-structure~1`
+`req~verify-changes_x.x.x.md-file~1`
 
-This plugin checks the repository structure during the build.
+PK verifies the content of the `doc/changes/changes_X.X.X.md` file for the project's current version.
 
-Rationale:
-As a build-breaker, the checks are automatically executed locally and in the CI process.
+Needs: req
+
+##### Verify Dependency Changes Section in changes_x.x.x.md File
+
+`req~verify-dependency-section-in-changes_x.x.x.md-file~1`
+
+PK verifies that the `## Dependency Updates` section in the `changes_X.X.X.md` file contains a list of the project dependencies that were added, updated or removed since the last projects release. PK formats the report like the following example:
```

Comment:
```suggestion
PK verifies that the `## Dependency Updates` section in the `changes_X.X.X.md` file contains a list of the project dependencies that were added, updated, or removed since the last release of the project. PK formats the report like the following example:
```